enhancements/oc/image-content-source-policy-awareness.md

```diff
@@ -0,0 +1,109 @@
+---
+title: image-content-source-policy-awareness
+authors:
+  - "@sallyom"
+reviewers:
+  - "@smarterclayton"
+  - "@soltysh"
+  - "@wking"
+approvers:
+  - "@smarterclayton"
+creation-date: 2020-05-19
+last-updated: 2020-05-19
+status: provisional
+---
+
+# ImageContentSourcePolicy (ICSP) Awareness
+
+ICSP allows OpenShift (CVO, CRI-O) to check down a list of possible mirrors to find an image with the matching digest it is
+looking for.  `oc` should do the same.  
+
+There have been several bugs opened around the experience of a 
+user in a disconnected environment using `oc adm release` commands.  If
+using a mirrored image and the mirrored source registry is connected, 
+the following commands do not succeed when in a disconnected environment:
+
+```console
+$ oc adm release extract --tools registry.example.com/repo/name:tag
+$ oc adm release mirror registry.example.com/repo/name:tag --to someregistry/repo/name
+$ oc adm must-gather
```

Comment:
It does, although it requires certain Image Streams to exist, so that's not a problem with functionality but with missing docs, tbh.